variables.tf

```diff
@@ -1,19 +1,28 @@
-variable iam_security_account_name {
-    description = "IAM Security Account Name"
-	default = "iam-security"
+variable "iam_security_account_name" {
+  description = "IAM Security Account Name"
+  default     = "iam-security"
 }
 
-variable project_name {
-    description = "Project prefix (aka license plate)."
+variable "project_name" {
+  description = "Project prefix (aka license plate)."
+}
+
+variable "project_config" {
```

Comment:
can we add type to this, and pas it around as an object instead of JSON string?

Reply:
It may be possible to specify it as an object in a way that all valid `project.json` files will validate as an object but the solution didn't come to me immediately. 

From here: https://www.terraform.io/docs/language/expressions/type-constraints.html

_Values that match the object type must contain all of the specified keys, and the value for each key must match its specified type. (Values with additional keys can still match an object type, but the extra attributes are discarded during type conversion.)_

Reply:
tl;dr I don't think it supports a flexible schema but I could be wrong